### ISSUE TITLE
Auto license-mode

### ISSUE THREAD

**Issue by Bluscream** [TeaSpeak/TeaSpeak#6]
It would be neat if you could provide another license mode for the config that calculates the license to show by the amount of slots being used (maybe even per virtual server if thats possible).

JS concept:
```js
calculateLicense(slots) {
  /* if (!internet.hasConnection()){
    return LicenseType.Offline;
  } */
  switch(slots) {
    case <= 32:
      return LicenseType.NoLicense;
    case <= 512:
      return LicenseType.NPL;
    default:
      return LicenseType.ATHP;
  } 
}

onClientConnectionInit(server, client) {
  if (settings.license == 5) {
    var slots = 0;
    for (var _server in servers) {
      slots += _server.slots;
    }
    license = calculateLicense(slots);
  } else if (settings.license == 6) {
    license = calculateLicense(server.slots);
}
  sendServerInit(client, license);
```

```yml
  # The displayed licence type to every TeaSpeak 3 Client
  # Available types:
  #  0: No licence
  #  1: Authorised TeaSpeak Host Provider License (ATHP)
  #  2: Offline/LAN License
  #  3: Non-Profit License (NPL)
  #  4: Unknown License
  #  5: Auto-License (Instance based)
  #  6: Auto-License (Server based)
  # This option is only for the premium version.
  licence: 5
```

**Comment by WolverinDEV** [comment]
Hey,
I really like this idea but there’s still one issue: You can’t change the license type while you’re lready connected.

But I would also implement it (I guess this evenings)

**Comment by Bluscream** [comment]
Hmm can't you forcefully send a serverupdate?

**Comment by WolverinDEV** [comment]
I can’t (already tested)

**Comment by Bluscream** [comment]
Dangit. Well, atleast reconnecting clients will get the new license then :)

**Comment by WolverinDEV** [comment]
Implemented in 1.0.10-Beta
